client/src/app/site/pages/meetings/pages/motions/components/motion-forward-dialog/services/motion-forward-dialog.service.ts

```diff
@@ -74,8 +81,18 @@ export class MotionForwardDialogService extends BaseDialogService<MotionForwardD
         const toMeetingIds = (await firstValueFrom(dialogRef.afterClosed())) as Ids;
         if (toMeetingIds) {
             try {
-                const forwardMotions = toForward.map(motion => this.formatService.formatMotionForForward(motion));
+                const motionIds = toForward.map(motion => motion.id);
+                await this.modelRequest.subscribeTo(getMotionForwardDataSubscriptionConfig(...motionIds));
+                await this.modelRequest.subscriptionGotData(
+                    getSpecificMotionSubscriptionName(MOTION_FORWARD_DATA_SUBSCRIPTION, motionIds)
+                );
+                const forwardMotions = toForward.map(motion =>
+                    this.formatService.formatMotionForForward(this.repo.getViewModel(motion.id))
+                );
                 const result = await this.repo.createForwarded(toMeetingIds, ...forwardMotions);
```

Comment:
That's a good idea, it is a bit bothersome though since I'll have to rewrite the function to work with the result of the fetch request instead of the view-models it uses currently. I suppose I'll try it, though.